languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -8660,6 +8660,11 @@ though, and has since been slightly modified)-->
             <example correction="C'est"><marker>Cet</marker> un garçon !</example>
         </rule>
         <rule id="CES_CEST" name="CES|SES au lieu de C'EST">
+            <antipattern>
+                <token regexp="yes">[cs]es</token>
+                <token regexp="yes">même</token>
+                <token postag="[JN].p" postag_regexp="yes"/>
```

Comment:
The test fails because the example sentence causes an error. That's because the `postag_regexp` should use `.*` instead of `.` (which matches only exactly one character).